### PR TITLE
Search book in home page

### PR DESCRIPTION
**Pull request by BinhNguyen3790**
Screenshot from 2019-04-08 14-33-11

https://edu-redmine.sun-asterisk.vn/issues/9868

### CODE REVIEW

**Comment by BinhNguyen3790** [comment]
ready

File: app/models/book.rb [hunk 1]
```diff
@@ -18,4 +18,12 @@ class Book < ApplicationRecord
     length: {maximum: Settings.book.pages}
   delegate :name, to: :author, prefix: :author
   scope :newest, ->{order created_at: :desc}
+
+  def self.search search
+    if search
+      self.where("name like ?", "%#{search}%")
+    else
+      self.all
+    end
```
Comment (VuThiTranVan) [inline app/models/book.rb]: viết scope hì
`where("name like ?", "%#{search}%") if search.present?`

File: app/views/layouts/_search.html.erb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<%= form_tag books_path, :method => "get" do %>
+  <div class="active-cyan-4 mb-4">
+    <%= text_field_tag :search, params[:search], class: "form-control",
+      placeholder: "Search" %>
```
Comment (VuThiTranVan) [inline app/views/layouts/_search.html.erb]: palceholer -> I18n

**Comment by BinhNguyen3790** [comment]
ready

File: app/views/layouts/_search.html.erb [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<%= form_tag books_path, :method => "get" do %>
```
Comment (VuThiTranVan) [inline app/views/layouts/_search.html.erb:1]: ` :method => "get" ` -> `method: :GET`

**Comment by VuThiTranVan** [comment]
Commented
có 1 comment, fix pull sau hì

**Pull request merged**